graylog2-server/src/main/java/org/graylog/plugins/views/search/permissions/UserStreams.java

```diff
@@ -40,10 +40,24 @@ public UserStreams(StreamPermissions streamPermissions, PermittedStreams permitt
         this.permittedStreams = permittedStreams;
     }
 
-    public ImmutableSet<String> loadAll() {
+    public ImmutableSet<String> loadAllMessageStreams() {
         return permittedStreams.load(streamPermissions);
```

Comment:
I'd suggest renaming the PermittedStreams#load to better represent what it does. I know it's not part of this PR, but when we are already touching the code, we could make it clearer.

Reply:
Good idea, thx!